jest.config.js

```diff
@@ -7,9 +7,13 @@ const supportsESMAndJestLightRunner = semver.satisfies(
   "^12.22 || ^13.7 || >=14.17"
 );
 const isPublishBundle = process.env.IS_PUBLISH;
+const debug = process.env.TEST_DEBUG;
 
 module.exports = {
-  runner: supportsESMAndJestLightRunner ? "jest-light-runner" : "jest-runner",
+  runner:
+    supportsESMAndJestLightRunner && !debug
+      ? "jest-light-runner"
```

Comment:
I think light runner is running in a Worker? I'm also having troubles when using `TEST_DEBUG` with a test label, it only seems to work when I use a describe label.

Reply:
https://github.com/babel/babel/pull/14488 adds `--runInBand` support.